src/action/list_directory.rs

```diff
@@ -56,20 +55,28 @@ impl From<Error> for session::Error {
     }
 }
 
+/// Contains information about an `UnsupportedValue` error.
```

Comment:
I mean, this doc string is not very useful. It still doesn't explain what an `UnsupportedValue` error is and in what circumstances can occur.

Reply:
Ok, a reason for that is probably my misunderstanding of this `struct` purpose. I think, it's just a wrap on a `tuple` for `UnsupportedValue` fields. That's why it was called `UnsupportedValueMessage` at first.

Reply:
I'm not sure, this issue is fixed.
But from my point of view this documentation string will make a person who reads it to find `UnsupportedValue` which is documented.